Use devcontainer

sverhoeven opened the pull request:
Stuck add inserting correctly formatted data

sverhoeven committed:
Use devcontainer

Stuck add inserting correctly formatted data

ermannolocascio commented:
no need to add manually. Use the webapp only as monitor or for truncating the sql database. Do the rest of the job (inserting) via python version.

sverhoeven committed:
Use older mariadb + mark state_history column as optional

As pulling latest gave timeouts
As Python script gave:
SQL Error: 1364 (HY000): Field 'state_history' doesn't have a default value

sverhoeven committed:
Point to main.py instead of curl
